server/home/home-alert/src/main/java/io/holoinsight/server/home/alert/service/calculate/AlertTaskCompute.java

```diff
@@ -139,21 +139,24 @@ private List<EventInfo> calculate(ComputeTaskPackage computeTaskPackage) {
             ComputeContext context = new ComputeContext();
             context.setTimestamp(computeTaskPackage.getTimestamp());
             context.setInspectConfig(inspectConfig);
-            EventInfo eventList = abstractUniformInspectRunningRule.eval(context);
-            if (eventList != null) {
-              eventLists.add(eventList);
+            EventInfo eventInfo = abstractUniformInspectRunningRule.eval(context);
+            if (eventInfo != null) {
+              eventLists.add(eventInfo);
             } else if (uniqueIds.contains(inspectConfig.getUniqueId())) {
-              eventList = new EventInfo();
-              eventList.setAlarmTime(computeTaskPackage.getTimestamp());
-              eventList.setUniqueId(inspectConfig.getUniqueId());
-              eventList.setIsRecover(true);
-              eventLists.add(eventList);
-              eventList.setEnvType(inspectConfig.getEnvType());
+              eventInfo = new EventInfo();
+              eventInfo.setAlarmTime(computeTaskPackage.getTimestamp());
+              eventInfo.setUniqueId(inspectConfig.getUniqueId());
+              eventInfo.setIsRecover(true);
+              eventLists.add(eventInfo);
+              eventInfo.setEnvType(inspectConfig.getEnvType());
             }
-            LOGGER.info("{} {} {} calculate package {} ,eventList: {}",
+            LOGGER.debug("{} {} {} calculate package {} ,eventList: {}",
```

Comment:
It is better to check LOGGER.isDebugEnabled() before LOGGER.debug() to reduce the cost of building json string.